src/web/routes/application/children-dob/set-children-in-session.js

```diff
@@ -0,0 +1,35 @@
+const { pickBy } = require('ramda')
+const { countKeysContainingString } = require('./count-keys')
+const { DAY_FIELD_SUFFIX } = require('./constants')
+
+const extractChildrenEntries = (val, key) => key.startsWith('child')
```

Comment:
Should this be `keyStartsWithChild` rather than `extraChildrenEntries` as it doesn't actually extract anything, its just the predicate?

Reply:
Yes very good @mikeSmith08. I noticed this previously but didn't update as it was existing code. Now might be a good time to change this.

Reply:
Turns out we already had a function for this `isChildEntry`